### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by vorpalhex** [vorpalhex/inktale#2]
IF content is by it's nature accessible. It'd be grand if the player was equally accessible. Spectre, the underlying CSS framework, should be fairly accessible already. Most of the concerns are story navigation and tab order. We may need to add some screen reader prompts know when we load in new menu panels that we've done crazy things to the DOM.
